src/core/stream_recv.c

```diff
@@ -162,6 +162,76 @@ QuicStreamRecvQueueFlush(
     }
 }
 
+//
+// Processes a received RELIABLE_RESET frame's payload.
+//
+_IRQL_requires_max_(PASSIVE_LEVEL)
+void
+QuicStreamProcessReliableResetFrame(
+    _In_ QUIC_STREAM* Stream,
+    _In_ QUIC_VAR_INT ErrorCode,
+    _In_ QUIC_VAR_INT ReliableOffset
+    )
+{
+    if (Stream->RecvMaxLength == 0 || ReliableOffset < Stream->RecvMaxLength) {
```

Comment:
Make sure to error out if we receive the frame, but haven't negotiated the extension.

Reply:
Noted.

Reply:
Done.